spec/models/order_detail_spec.rb

```diff
@@ -2017,4 +2017,27 @@ def set_cancellation_cost_for_all_policies(cost)
     end
 
   end
+
+  describe "#reconcile_if_skip_review" do
+    before do
+      order_detail.actual_cost = 10
+      order_detail.actual_subsidy = 0
```

Comment:
Look into a spec when these aren't set. Look at how this can happen.

Make a model and system spec.